benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(SUBSYS_NAME benchmarks)
+set(SUBSYS_DESC "Point cloud library benchmarks")
+set(SUBSYS_DEPS common features search kdtree io)
+set(DEFAULT OFF)
+set(build TRUE)
+set(REASON "Disabled by default")
+PCL_SUBSYS_OPTION(build "${SUBSYS_NAME}" "${SUBSYS_DESC}" ${DEFAULT} "${REASON}")
+PCL_SUBSYS_DEPEND(build "${SUBSYS_NAME}" DEPS ${SUBSYS_DEPS})
+if(NOT build)
+  return()
+endif()
+
+find_package(benchmark REQUIRED)
+add_custom_target(run_benchmarks)
+
+macro(PCL_ADD_BENCHMARK _name)
```

Comment:
Move this macro to where the other macro is, ie:
https://github.com/PointCloudLibrary/pcl/blob/master/cmake/pcl_targets.cmake ?